tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -3501,7 +3506,15 @@ def get_valid_tactics(
                         self.expert_count,
                         self.weight_per_expert,
                 ):
-                    tactics.append((mma_tiler_mn, cluster_shape_mn))
+                    for split_k in split_k_candidates:
+                        # K-tiles must be evenly divisible by split_k,
+                        # and each split must contain whole experts.
+                        k_tiles = k // _MMA_TILE_K
+                        tiles_per_expert = self.weight_per_expert // _MMA_TILE_K
+                        if (k_tiles % split_k == 0 and
+                            (k_tiles // split_k) % tiles_per_expert == 0):
```

Flake8 (7.3.0)

[error] visually indented line with same indent as next logical line

(E129)